analysis.tf

```diff
@@ -0,0 +1,136 @@
+resource "aws_instance" "analysis" {
+  key_name                    = "${var.key_name}"
+  ami                         = "${data.aws_ami.rhel_67.id}"
+  instance_type               = "r4.4xlarge"
```

Comment:
Please use t2.micro to match HAProxy

Reply:
Good catch @david-wirelab - however to match SG we should use c5.xlarge (4CPU, 8GB)
I know it's way to much - but I want to be as close to SG as possible

Reply:
Ok, fine.